doc/man3/SSL_new_domain.pod

```diff
@@ -0,0 +1,156 @@
+=pod
+
+=head1 NAME
+
+SSL_new_domain,
+SSL_is_domain,
+SSL_get0_domain,
+SSL_get_domain_flags,
+SSL_CTX_set_domain_flags,
+SSL_CTX_get_domain_flags,
+SSL_DOMAIN_FLAG_SINGLE_THREAD,
+SSL_DOMAIN_FLAG_MULTI_THREAD,
+SSL_DOMAIN_FLAG_THREAD_ASSISTED,
+SSL_DOMAIN_FLAG_BLOCKING,
+SSL_DOMAIN_FLAG_LEGACY_BLOCKING - SSL object interface for managing QUIC event domains
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_CTX_set_domain_flags(SSL_CTX *ctx, uint64_t domain_flags);
+ int SSL_CTX_get_domain_flags(SSL_CTX *ctx, uint64_t *domain_flags);
+
+ SSL *SSL_new_domain(SSL_CTX *ctx, uint64_t flags);
+
+ int SSL_is_domain(SSL *ssl);
+ SSL *SSL_get0_domain(SSL *ssl);
+
+ int SSL_get_domain_flags(SSL *ssl, uint64_t *domain_flags);
+
+ #define SSL_DOMAIN_FLAG_SINGLE_THREAD
+ #define SSL_DOMAIN_FLAG_MULTI_THREAD
+ #define SSL_DOMAIN_FLAG_THREAD_ASSISTED
+ #define SSL_DOMAIN_FLAG_BLOCKING
+ #define SSL_DOMAIN_FLAG_LEGACY_BLOCKING
+
+=head1 DESCRIPTION
+
+The SSL_new_domain() function creates a new QUIC event domain, represented as an
+SSL object. This is known as a QUIC domain SSL object (QDSO). The concept of a
+QUIC event domain is discussed in detail in L<openssl-quic-concurrency(7)>.
+
+A QUIC domain SSL object can be managed in the same way as any other SSL object,
+in that it can be refcounted and freed normally. A QUIC domain SSL object is the
+parent of a number of child objects such as QUIC listener SSL objects. Once a
+QUIC domain SSL object has been created, a listener can be created under it
+using L<SSL_new_listener_from(3)>.
+
+SSL_is_domain() returns 1 if a SSL object is a QUIC domain SSL object.
+
+SSL_get0_domain() obtains a pointer to the QUIC domain SSL object in a SSL
+object hierarchy (if any).
+
+=head2 Domain Flags
+
+Domain flags are a set of flags which configure the concurrency model used by
+the OpenSSL QUIC implementation. For more information, including the meanings of
+the B<SSL_DOMAIN_FLAG> values, see L<openssl-quic-concurrency(7)>.
+
+SSL_CTX_set_domain_flags() and SSL_CTX_get_domain_flags() can be used to set and
+get the domain flags configured on a B<SSL_CTX> which is using a QUIC
```

Comment:
**an** SSL_CTX. Here and below.

Reply:
Those changes seem to be no longer part of stuff I've pulled in.